src/tools/illink/src/ILLink.Tasks/build/Microsoft.NET.ILLink.targets

```diff
@@ -55,6 +55,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <_ComObjectDescriptorSupport Condition="'$(_ComObjectDescriptorSupport)' == ''">false</_ComObjectDescriptorSupport>
     <_DesignerHostSupport Condition="'$(_DesignerHostSupport)' == ''">false</_DesignerHostSupport>
     <_DefaultValueAttributeSupport Condition="'$(_DefaultValueAttributeSupport)' == ''">false</_DefaultValueAttributeSupport>
+    <DynamicCodeSupport Condition="'$(DynamicCodeSupport)' == ''">true</DynamicCodeSupport>
```

Comment:
What's the ordering between this and 

https://github.com/dotnet/runtime/blob/a0b8890f989d2d4b37507b99d94249506f161d6f/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets#L42

Which gets imported first? Cuz whichever one does, will win. So if the Native.targets get imported first, that would be best.

Reply:
Looks like the native AOT targets get imported first. So that's good.

https://github.com/dotnet/sdk/blob/6e4637f352d046e599d05b9090946ff818b320f1/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets#L1331-L1332

I would hope some test would fail if these ever got switched around.

Reply:
Things would not behave as expected if this is set to true. We error out if someone sets it to true: https://github.com/dotnet/runtime/blob/5d1981ef2d3917bc65bba9dbd36d3193f91825c5/src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets#L58